languages/ruby/exercises/concept/strings/.docs/introduction.md

```diff
@@ -0,0 +1,4 @@
+# Introduction
+
+A `String` in Ruby is an object that holds and manipulates an arbitrary sequence of bytes, typically representing characters.
```

Comment:
I'd also add "Strings are manipulated by calling the string's methods." or something, as this is different to many functional languages.